java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -613,4 +614,79 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.
+     *
+     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets.
+     * @param options Weight and Aggregate options.
+     * @return The resulting sorted set from the union.
+     * @example
+     *     <pre>{@code
+     * WeightAggregateOptions options =
+     *     WeightAggregateOptions.builder()
+     *             .aggregate(Aggregate.MAX)
+     *             .weights(List.of(1.0, 2.0))
+     *             .build();
+     * String[] payload = client.zunionstore("newSortedSet", new String[] {"mySortedSet1", "mySortedSet2"}, options).get()
+     * assert payload.equals(new String[] {"elem1", "elem2", "elem3"});
+     * }</pre>
+     */
+    CompletableFuture<String[]> zunion(String[] keys, WeightAggregateOptions options);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.<br>
+     * To perform a zunion operation while specifying custom weights and aggregation settings, use
+     * {@link #zunion(String[], WeightAggregateOptions)}
+     *
+     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets.
+     * @return The resulting sorted set from the union.
+     * @example
+     *     <pre>{@code
+     * String[] payload = client.zunionstore("newSortedSet", new String[] {"mySortedSet1", "mySortedSet2"}).get()
+     * assert payload.equals(new String[] {"elem1", "elem2", "elem3"});
+     * }</pre>
+     */
+    CompletableFuture<String[]> zunion(String[] keys);
+
+    /**
+     * Returns the union of members and their scores from sorted sets specified by the given <code>
+     * keys</code>.
+     *
+     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets.
+     * @param options Weight and Aggregate options.
+     * @return The resulting sorted set from the union with their scores.
+     * @example
+     *     <pre>{@code
+     * WeightAggregateOptions options =
+     *     WeightAggregateOptions.builder()
+     *             .aggregate(Aggregate.MAX)
+     *             .weights(List.of(1.0, 2.0))
+     *             .build();
+     * Map<String, Double> payload = client.zunionstoreWithScores("newSortedSet", new String[] {"mySortedSet1", "mySortedSet2"}, options).get()
+     * assert payload.equals(Map.of("elem1", 1.0, "elem2", 2.0, "elem3", 3.0));
+     * }</pre>
+     */
+    CompletableFuture<Map<String, Double>> zunionWithScores(
+            String[] keys, WeightAggregateOptions options);
+
+    /**
+     * Returns the union of members and their scores from sorted sets specified by the given <code>
+     * keys</code>.<br>
+     * To perform a zunionWithScores operation while specifying custom weights and aggregation
```

Comment:
wrap `zunionWithScores` with `code` or replace with a synonim ("To perform the same operation...")